src/ui/components/shared/ReplayLogo.tsx

```diff
@@ -9,7 +9,7 @@ const logoSizes = {
 const logoColors = {
   white: "#FFF",
   fuschia: "#F02D5E",
-  gray: "var(--cool-gray-500)",
+  gray: "#6b7280)",
```

Comment:
Revert this specific one.

Reply:
There's no benefit to it being a variable.

* It doesn't change from dark/light theme
* We don't use it anywhere else

If it were a variable named logo, I could see us using it in the future and reserving it. But it's an orphan variable we're trying to remove.

Reply:
Never mind. I don't want to come up with a short term fake variable yet but I'll do it anyway.